### DISCUSSION TITLE
How to document a transaction record for FMNH Exhibitions.

### DISCUSSION THREAD

**Discussion by fmjjones** [fieldmuseum/EMu-Documentation#688] [HOW TO...]
Loans to Exhibitions - You must attach the Exhibition Event record to the transaction record AND add "Exhibitions" to the Department field on the Security tab.

This allows Exhibitions and Conservation staff to display and attach your transaction to any Event, Exhibit Object or Conservation record necessary for use in an exhibit.

**Comment by mariegeorg** [comment]
There are several phases of activity in EMu in each exhibition project where Exhibitions interacts with the collections' catalog records.  Early on, we are in the **Object List Building** phase.  This is the time period when exhibition developers and curators may be visiting the collections looking for candidate objects for the exhibition.  Not all of these objects may ultimately be part of the exhibition, but for objects that have good potential or interest (not literally everything we saw,) we do need for EMu catalog records to exist in order for us to move on to the next phase.  We also need for individual part records to exist because those parts will get individual documentation.

Exhibition developers attach those records to the Event record in order to spur the creation of Exhibition Object (ExhObj) records.  The existence of the Exhibition Object record is required for the **Object Documentation** phase.  In this phase, we record in the ExhObj records  information about the (potential) installation of the object including level of interest, where it fits in the story, notes about the display, measurements, photos, and all information designers need to make their drawings. We also need this information to manage requesting conservation assessments, making loan requests, and even budgeting for the exhibition production.  (Every object costs time and therefore money.)  

We CANNOT make exhibition object records except by attaching an existing catalog record (parent or child) to our Event record.  We therefore depend on collection managers to create records or part records in order for us to do ANYTHING to officially consider using the object.  In fact, the rest of the team will have no idea which objects to assess or document if the object list building process is incomplete because those catalog records do not exist. The part records are particularly important so that we can specify which elements are to be included (i.e. part or counterpart or both, which preps, which bones) and document and track them individually.

Once an object has been documented, passed conservation assessment, and physically fits in the final story and design (and case layouts are signed off) the exhibition team move from design to production. Only at that point Registration will facilitate the actual object **Loan Transactions** along with the physical transfer, referring to the finalized list.  Months of work with the records leads up to the loan transaction.

Over time, an catalog record can be attached to many Events, spinning off multiple ExhObj records, each documenting a different installation.  This history can be seen in Catalog in both the Events and ExhObj tabs.  The images we produce are searchable in the Multimedia module by both exhibition inventory number (ABC-###) and your catalog number and can be attached directly to your catalog records.
